scope/src/shared/grouped_body.jinja

```diff
@@ -77,7 +77,7 @@ Output:
 | Name | Value |
 |---|---|
 {% for data in group.additionalData %}
-|{{ data.name }}|{% if data.output %}`{{ data.output }}`{% endif %}|
+|{{ data.name }}|{% if data.output %}`{{ data.output|split("\n")|map("trim")|join(", ") }}`{% endif %}|
```

Comment:
Instead of using replacing `\n` with a `, `, why don't we replace it with `<br>`? Then it will render correctly in markdown (with multiple lines)

Reply:
Great suggestion! Multi line output looks a lot better like this:
![Screenshot 2024-06-16 at 3 28 25 PM](https://github.com/oscope-dev/scope/assets/1007983/0f16df7a-1578-4e78-aa66-c2dfcff36da3)